starters/serverless-framework-sqs-dynamodb/src/models/technology/create.ts

```diff
@@ -0,0 +1,19 @@
+import { v4 as uuidv4 } from 'uuid';
+import { TechnologyCreate } from '@/types/technology';
+import { putItem } from '@/utils/dynamodb/putItem';
+import { addToCache } from '@/utils/cache/addToCache';
+import { getCacheKey } from './getCacheKey';
+
+export const create = async (payload: TechnologyCreate) => {
+	const newTechnology = {
+		id: uuidv4(),
+		...payload,
+	};
+	const didPersist = await putItem(process.env.TECHNOLOGIES_TABLE, newTechnology);
+	if (didPersist) {
+		await addToCache(getCacheKey(newTechnology.id), newTechnology);
+		return newTechnology;
+	}
+
+	return null;
```

Comment:
Do we suppress an error here? If yes, shouldn't we log it? Should this `return null` ever be reached? 🤔

Reply:
No error suppression so this is good as is.

Reply:
I was now able to put together a mental model of the structure and understand that this is not the handler, but a level down.